Hippo.Web/Controllers/AccountController.cs

```diff
@@ -158,17 +159,39 @@ public async Task<ActionResult> Create([FromBody] CreateModel model)
         {
             return BadRequest("Missing SSH Key");
         }
-        if (!model.SshKey.StartsWith("-----BEGIN RSA PRIVATE KEY-----") || !model.SshKey.EndsWith("-----END RSA PRIVATE KEY-----"))
+        if (!model.SshKey.StartsWith("ssh-") || model.SshKey.Trim().Length < 25)
         {
             return BadRequest("Invalid SSH key");
         }
 
+        var sponsorAccount = await _dbContext.Accounts.Include(a => a.Owner).SingleAsync(a => a.Id == model.SponsorId);
+
+
+        var sb = new StringBuilder();
+        sb.AppendLine("sponsor:");
+        sb.AppendLine($"    accountname: {sponsorAccount.Name}");
+        sb.AppendLine($"    name: {sponsorAccount.Owner.Name}");
+        sb.AppendLine($"    email: {sponsorAccount.Owner.Email}");
+        sb.AppendLine($"    kerb: {sponsorAccount.Owner.Kerberos}");
+        sb.AppendLine($"    iam: {sponsorAccount.Owner.Iam}");
+        sb.AppendLine($"    mothra: {sponsorAccount.Owner.MothraId}"); 
+        sb.AppendLine();
+        sb.AppendLine("Account:");
+        sb.AppendLine($"    name: {currentUser.Name}");
+        sb.AppendLine($"    email: {currentUser.Email}");
+        sb.AppendLine($"    kerb: {currentUser.Kerberos}");
+        sb.AppendLine($"    iam: {currentUser.Iam}");
+        sb.AppendLine($"    mothra: {currentUser.MothraId}"); 
+        sb.AppendLine($"    key: {model.SshKey}");
```

Comment:
It's hard to tell from the screenshot if there are line breaks in the `key` property or if it is just wrapping. If there are line breaks, then it's not valid yaml. If it is a single line, you'll want to enclose it in double-quotes to prevent any special characters from being interpreted as yaml string-formatting directives.

A double-quoted string is the only way to have a block-formatted yaml string while preventing the yaml parser from inserting whitespace. This...
```yaml
key : "BEGIN

  abc\
  123\
  @#$%

  END"
```
Will result in a string of `"BEGIN\nabc123@#$%\nEND"`

You can test yaml at https://yaml-online-parser.appspot.com/
[Here's an exhaustive explanation of yaml string formatting](https://stackoverflow.com/questions/3790454/how-do-i-break-a-string-in-yaml-over-multiple-lines)

Reply:
The line breaks from my first example were because that was how I pasted it in.
Although, I suppose we could validate if they pasted them in...

Reply:
As long as it passes that yml validation i'm good with it.  does it work w/ your example?  can't tell about the line breaks, and I don't think you need to quite multi-line in YAML but it's worth trying if the first version fails and you get stuck